ecosystem/sep-0006.md

```diff
@@ -110,8 +116,90 @@ Example:
 }
 ```
 
-Every other HTTP status code will be considered an error. The body should contain error details. 
+Every other HTTP status code will be considered an error. The body should contain error details.
 For example:
+
+```json
+{
+   "error": "This anchor doesn't support the given currency code: ETH"
+}
+```
+
+### Transaction History
+
+The transaction history endpoint helps anchors enable a better experience for users using an external wallet. With it, wallets can display the status of deposits and withdrawals while they process and a history of past transactions with the anchor. It's only for transactions that are deposits to or withdrawals from the anchor.
+
+Endpoint: `DEPOSIT_SERVER/transactions`<br>
+Purpose: Get status of current and past deposits/withdrawals.<br>
+Method: GET<br>
+Request parameters
+
+Name | Type | Description
+-----|------|------------
+`asset_code` | string | The code of the asset of interest. E.g. BTC,ETH,USD,INR,etc
+`account` | string | The stellar account ID involved in the transactions
+`no_older_than` | UTC ISO 8601 string | (optional) The response should contain transactions starting on or after this date & time
+`n` | int | (optional) the response should contain at most `n` transactions
+`paging_id` | string | (optional) the response should contain transactions starting prior to this ID (exclusive)
+
+On success the endpoint should return `200 OK` HTTP status code and a JSON object with the following fields:
+
+Name | Type | Description
+-----|------|------------
+`transactions` | array | List of transactions as requested by the client, sorted in time-descending order
+
+Each object in the `transactions` array should have the following fields:
+
+Name | Type | Description
+-----|------|------------
+`id` | string | Unique, anchor-generated id for the deposit/withdrawal
+`kind` | string | `deposit` or `withdrawal`
+`status` | string | Processing status of deposit/withdrawal
+`status_eta` | number | (optional) Estimated number of seconds until a status change is expected
+`amount` | float | (optional) Amount of deposit/withdrawal
+`started_at` | UTC ISO 8601 string | (optional) start date and time of transaction
+`completed_at` | UTC ISO 8601 string | (optional) completion date and time of transaction
+`stellar_operation_id` | string | (optional) operation_id on Stellar network of transfer that either completed the deposit or started the withdrawal
```

Comment:
`operation_id` is specific to Horizon. I think the better way would be to use something like `[transaction_id]-[operation_id_in_transaction]`, ex. `3389e9f0f1a65f19736cacf544c2e825313e8447f569233bb8db39aa607c8889-0`.

Reply:
Its probably fine to just use txid here right?